internal/xds/translator/testdata/out/ratelimit-config/empty-header-matches.yaml

```diff
@@ -1,13 +1,19 @@
 domain: first-listener
 descriptors:
-  - key: first-route-key-rule-0-match--1
-    value: first-route-value-rule-0-match--1
-    rate_limit:
-      requests_per_unit: 5
-      unit: SECOND
-      unlimited: false
-      name: ""
-      replaces: []
-    descriptors: []
+  - key: key-first-route
+    value: value-first-route
+    rate_limit: null
+    descriptors:
+      - key: key-rule-0-match--1
```

Comment:
The same as above, slightly prefer just the rule and match index part, no key or value prefix.

Reply:
removed